x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/categorization_job_creator.ts

```diff
@@ -41,6 +41,7 @@ export class CategorizationJobCreator extends JobCreator {
     ML_JOB_AGGREGATION.COUNT;
   private _categorizationAnalyzer: CategorizationAnalyzer = {};
   private _defaultCategorizationAnalyzer: CategorizationAnalyzer;
+  private _partitionField: string | null = null;
```

Comment:
in the `MultiMetricJobCreator` we store the actual `Field` object for the split field for use later in the UI rather than just its name.
Storing the field name here is fine, but to be explicit about the difference, perhaps this should be called `_partitionFieldName`

Reply:
Updated here https://github.com/elastic/kibana/pull/75061/commits/9cc12a0717ab77c24549485cf040f991b41329f1

Reply:
Updated here https://github.com/elastic/kibana/pull/75061/commits/4b2f2e01b24ae30e0bd8a61ddf877015e5785ecb